modules/installation-configure-proxy.adoc

```diff
@@ -39,7 +39,7 @@ endif::bare-metal[]
 +
 [NOTE]
 ====
-The Proxy object's `status.noProxy` field is populated by default with the instance metadata endpoint (`169.254.169.254`) and with the values of the `networking.machineCIDR`, `networking.clusterNetwork.cidr`, and `networking.serviceNetwork` fields from your installation configuration.
+The Proxy object's `status.noProxy` field is populated by default with the instance metadata endpoint (`169.254.169.254`) and with the values of the `networking.machineNetwork[].cidr`, `networking.machineNetwork[].cidr`, and `networking.serviceNetwork` fields from your installation configuration.
```

Comment:
Two `networking.machineNetwork[].cidr` here? duplicated? one of them should be `networking.clusterNetwork[].cidr`, right?

Reply:
@jianlinliu, oops, I'll fix that!